manager/src/main/java/com/redhat/service/bridge/manager/workers/id/WorkerIdProvider.java

```diff
@@ -4,13 +4,33 @@
 
 import javax.enterprise.inject.Produces;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
 public class WorkerIdProvider {
 
-    private static final String WORKER_ID = UUID.randomUUID().toString();
+    private static final String RANDOM_STATIC_WORKER_ID = UUID.randomUUID().toString();
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WorkerIdProvider.class);
+
+    private String findPodNameFromKubernetes() {
```

Comment:
Nitpick: This isn't really anything to do with Kubernetes, so should we name it a bit better e.g `findWorkerNameFromHost`?